### ISSUE TITLE
Failed to parse CLI options; expected argument for flag `/c

### ISSUE THREAD

**Issue by papanito** [terraform-linters/tflint#2225]
### Summary

When I pass `-c` on tflint running on a bash shell in windows I get

```
tflint -c "/cygdrive/d/workspace/demo/.tflint.hcl" --recursive
Failed to parse CLI options; expected argument for flag `/c, /config', but got option `/cygdrive/d/workspace/demo/.tflint.hcl'
```

### Command

tflint -c "/cygdrive/d/workspace/demo/.tflint.hcl"

### Terraform Configuration

```terraform
N/A
```

### TFLint Configuration

```HCL
N/A
```

### Output

```Shell
Failed to parse CLI options; expected argument for flag `/c, /config', but got option `/cygdrive/d/workspace/demo/.tflint.hcl'
```

### TFLint Version

0.55.1

### Operating System

- [ ] Linux
- [ ] macOS
- [x] Windows
